docs/apis-tools/frontend-development/03-forms/02-embed-forms/01-concepts.md

```diff
@@ -0,0 +1,148 @@
+
+<img src={FormPlaygroundImg} alt="Camunda Forms playground" />
+
+<div style={{marginTop: '24px', marginBottom: '8px'}}>
+   <a
+      className={clsx(
+         "button button--outline button--secondary button--lg"
+      )}
+      href="https://camunda-form-playground.netlify.app/">
+      Try form playground
+   </a>
+</div>
+
+Try the form playground yourself directly on the Web—no login needed.
+
+## The form schema
+
+A form is serialized as plain JSON with a simple, flat structure to maximize flexibility and versatility. In the root, a form contains some metadata attributes. The main form is defined by a list of components, where the components carry their layout properties themselves (e.g. which row a component belongs to). This is in contrast to markup languages such as HTML, where the arrangement of the nodes determines the layout. This enables backward compatibility and compatibility with user-defined renderers. 
+
+See this simple form schema for example, and the resulting form:
+
+```json
+{
+  "components": [
+    {
+      "label": "First name",
+      "type": "textfield",
+      "layout": {
+        "row": "Row_0hqc9xn",
+        "columns": null
+      },
+      "id": "Field_05l2s7c",
+      "key": "firstName"
+    },
+    {
+      "label": "Last name",
+      "type": "textfield",
+      "layout": {
+        "row": "Row_0hqc9xn",
+        "columns": null
+      },
+      "id": "Field_0nw7e1c",
+      "key": "lastName"
+    },
+    {
+      "label": "Income",
+      "type": "number",
+      "layout": {
+        "row": "Row_1ggwq2d",
+        "columns": 8
+      },
+      "id": "Field_12yshuy",
+      "key": "monthlyNetIncome",
+      "description": "Monthly net income",
+      "appearance": {
+        "prefixAdorner": "USD"
+      },
+      "increment": "100",
+      "validate": {
+        "required": true,
+        "min": 0
+      }
+    }
+  ],
+  "type": "default",
+  "id": "ExampleForm",
+  "executionPlatform": "Camunda Cloud",
+  "executionPlatformVersion": "8.4.0",
+  "exporter": {
+    "name": "Camunda Modeler",
+    "version": "5.18.0"
+  },
+  "schemaVersion": 12
+}
+```
+
+<FormViewer schema={ simpleForm } />
+
+All form-js packages share the same [JSON schema](https://github.com/bpmn-io/form-js/tree/develop/packages/form-json-schema) <GHIcon /> for forms.
+
+This enables the interoperability of the created forms between the form editor and the viewer and possibly also between custom-made form renderers, or translating from a Camunda Form to another form. Using the form schema, you can write extensions to existing components, while still receiving benefits from updates made to the core form-js libraries.
+
+The schema abstracts the view model from the viewer, and allows you to inject another expression or templating language as an alternative to FEEL, since expressions are simply stored as strings.
```

Comment:
We're using a bit of MVVM language here, but I don't know if we are doing it properly. Actually I'd argue we don't have a true view-model view separation at the moment. View-model defines UI behaviors already, the schema is more of just a model.